subsys/net/lib/http/iterables.ld

```diff
@@ -0,0 +1 @@
+ITERABLE_SECTION_ROM(http_content_type, 4)
```

Comment:
Please add copyright + spdx info to this file.
Also related to change in commit dacb3dbfeb5d34ffc02c2e2f547dbd569ace008e,
please add this before the macro call.
```
#include <zephyr/linker/iterable_sections.h>
```